From: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
Date: Wed, 11 Mar 2026 12:53:17 +0100
Subject: [PATCH 06/13] parser: name ns_domain and network_v9 classes

Replace placeholder class table entries unknown8 and unknown15 with their intended names in aa_class_table.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>
(cherry picked from commit a10366e9b8e04dc6d09a9f1713993197fe9f4272)
Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 parser/rule.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/parser/rule.cc b/parser/rule.cc
index db89447..2c10612 100644
--- a/parser/rule.cc
+++ b/parser/rule.cc
@@ -28,14 +28,14 @@ const char *aa_class_table[] = {
 	"rlimit",
 	"domain",
 	"mount",
-	"unknown8",
+	"ns_domain",
 	"ptrace",
 	"signal",
 	"xmatch",
 	"env",
 	"argv",
 	"network",
-	"unknown15",
+	"network_v9",
 	"label",
 	"mqueue",
 	"mqueue",
